src/extra/gpkg.jl

```diff
@@ -0,0 +1,215 @@
+UNKNOWN_SRID = -2
+DEFAULT_SRID = 0
+
+# Requirement 2: contains "GPKG" in ASCII in "application_id" field of SQLite db header
+# Reminder: We have to set this (on-write) after there's some content
+# so the database file is not zero length
```

Comment:
Please add URLs to specific parts of the GPKG specification whenever possible. That helps with providing better review comments and also helps future maintainers of the code.